CSharp/Library/Microsoft.Bot.Connector.Shared/MicrosoftAppCredentials.cs

```diff
@@ -139,15 +140,21 @@ public static void TrustServiceUrl(string serviceUrl, DateTime expirationTime =
         {
             try
             {
+                var scopeHashset = new ConcurrentDictionary<string, byte>();

```

Comment:
This is not a hashset anymore, can we just name it "scopes" so it is not confusing?

Reply:
changed to "scopes"